src/main/webapp/search-script.js

```diff
@@ -0,0 +1,129 @@
+function collectUnitsFromSearch() {
```

Comment:
wait why is this here? sorry if i'm missing something but does this need to be in your PR?

Reply:
By accident deleted that file so had to add it back, sorry about that!